wordpress-mobile/WordPress-iOS
# Fixes spinning wheel on top of blog list.

@frosty noticed an issue with our blog list, that's making the pull-to-refresh spinning wheel to come up on top of our table view.

This PR fixes the issue.

## To test:

1. Launch the app.
2. Tap "Switch Site" in the top right corner.

Make sure you don't see this:

![](https://user-images.githubusercontent.com/4780/96598726-1260a100-12e7-11eb-9241-fb86a8a4696b.png)

## PR submission checklist:
